cocos2d/CCScheduler.m

```diff
@@ -137,19 +300,20 @@ -(void)setPaused:(BOOL)paused
 @end
 
 
-@interface NSNull(CCSchedulerTarget)<CCSchedulerTarget>
+@interface NSNull(CCSchedulableTarget)<CCSchedulableTarget>
 @end
 
 
-@implementation NSNull(CCSchedulerTarget)
+@implementation NSNull(CCSchedulableTarget)
 -(NSInteger)priority {return NSIntegerMax;}
 @end
 
 
+//MARK: CCTimer
```

Comment:
Hm, I initially agreed, but if I'm going to find/replace them all, that should be a separate pull request. I don't want to delay this or make the pull request even bigger.